packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Add and configure packages
+---
+Next, you need to add Okta to your application by installing and configuring our SDK.
+
+You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/configure-packages/) guide.
+
```

Comment:
We need a new para here (and a stack selector) that shows how to install the additional package (Authentication SDK) required for building custom sign-in UI, on top of the package that was already installed by following the previous guide. For example:

> To build custom sign-in UI,  you need to install one additional package: the Okta Authentication SDK. This SDK works together with the OIDC SDK you already have installed to make authentication requests to Okta from your own views.
> <stack selector: ios/android>

Reply:
The content in this stack selector should show:
* How to install the package
* How to configure the package/initialize the SDK in code

The first few paras of `Use the Okta Java Authentication SDK` from the primary-authentication step should be moved here as the Android content. @IldarAbdullin-okta can provide the iOS content.

Reply:
I added a new paragraph, created the folders for the stack selector, added notes on what the stack selector should show in the iOS file and moved the install okta auth sdk content from the Android primary-authentication step into the new Android file.

Could @FeiChen-okta please check the changes for Android?
@nbarbettini - please verify that the changes are what you wanted.
@IldarAbdullin-okta - can you please provide the iOS content for the install okta auth sdk?

Reply:
@susanharper-okta , sure, will submit into this branch

Reply:
@susanharper-okta The android part is correct.

Reply:
Double checked iOS part. Changes are there